src/global.rs

```diff
@@ -165,6 +165,75 @@ pub mod print {
         write::h3(&mut GlobalWriter, s);
     }
 
+    /// Announce the name of a buildpack
+    ///
+    /// Use together with [all_done]
+    ///
+    /// ```
+    #[doc = include_str!("./docs/global_setup.rs")]
+    ///
+    /// let started = print::buildpack("Heroku Awesome Buildpack");
+    /// print::bullet("Just add awesome.");
+    /// print::all_done(&Some(started));
+    #[doc = include_str!("./docs/global_done_one.rs")]
+    /// ### Heroku Awesome Buildpack
+    ///
+    /// - Just add awesome.
+    /// - Done (finished in < 0.1s)
+    #[doc = include_str!("./docs/global_done_two.rs")]
+    /// ```
+    pub fn buildpack(s: impl AsRef<str>) -> Instant {
+        write::h2(&mut GlobalWriter, s);
+        Instant::now()
+    }
+
+    /// Header to break up subsections in a buildpack's output
+    ///
+    /// ```
+    #[doc = include_str!("./docs/global_setup.rs")]
+    ///
+    /// let started = print::buildpack("FYEO Buildpack");
```

Comment:
Couldn't think of a better header/list example, huh? 😅

Reply:
It's better than foo/bar 😄 I put a bunch of roller coaster tycoon references in the tests, Manuel was reviewing my PRs.